fedora/SPECS/pam_usb.spec

```diff
@@ -14,6 +14,7 @@ Version:   %{version}
 Release:   %{release}
 Prefix:    /usr
 Group:     System Environment/Base
+BuildRequires: rsync libudisks2-devel libxml2-devel
```

Comment:
Why rsync?

Reply:
Because the rsync command is called in the fedora spec @mcdope .

Reply:
Where? Not as far as I know.

Reply:
nvm, forgot about that - sorry :D 

But actually, we should change that to use regular cp instead. I don't see the need to use rsync there.

Reply:
Gonna accept the change and will create a ticket to get rid of rsync for that.